docs/index.md

```diff
@@ -67,6 +63,10 @@ Chisel is a member of the Ubuntu family. It’s an open source project that warm
 
 Thinking about using Chisel for your next project? <a href="https://canonical.com/#get-in-touch#">Get in touch!</a>
 
+---------
+
+This documentation uses the [Diátaxis documentation structure](https://diataxis.fr/).
```

Comment:
Thanks for being willing to try this out.

This looks good enough for now. But I'd like to find a better place for it still.

Just continue discussing the following on the side:
 - In the rockcraft docs, there's a special "docs contributions" page (https://documentation.ubuntu.com/rockcraft/stable/contribute-to-this-documentation/). Is that part of the template? If we're having one as well for Chisel, isn't that the obvious place to have this reference? TBC